Source/Core/Common/Timer.h

```diff
@@ -29,6 +29,7 @@ class Timer
 	static double GetDoubleTime();
 
 	static std::string GetTimeFormatted();
+	static std::string GetDateTimeFormatted(double time);
```

Comment:
Would be nice to have documentation as to what specifically "time" represents, since it's not really clear how a time can be expressed in a "double".